inc/az_ulib_ipc_api.h

```diff
@@ -360,6 +362,103 @@ AZ_NODISCARD az_result az_ulib_ipc_call_with_str(
     az_span model_in_span,
     az_span* model_out_span);
 
+/**
+ * @brief   Get value of a published property.
+ *
+ * @param[in]   interface_handle  The #az_ulib_ipc_interface_handle with the interface handle. It
+ *                                cannot be `NULL`. Call az_ulib_ipc_try_get_interface() to get
+ *                                the interface handle.
+ * @param[in]   property_index    The #az_ulib_capability_index with the property handle.
+ * @param[out]  model_out         The `az_ulib_model_out` that points to the memory where the
+ *                                capability should store the property value.
+ *
+ * @pre     IPC shall already be initialized.
+ * @pre     \p interface_handle shall not be 'NULL'.
+ *
+ * @return The #az_result with the result of the get.
+ *  @retval #AZ_OK                              If the IPC get success getting the property.
+ *  @retval #AZ_ERROR_ITEM_NOT_FOUND            If the target interface does not exist.
+ *  @retval Others                              Defined by the target get.
+ */
+AZ_NODISCARD az_result az_ulib_ipc_get(
```

Comment:
>az_ulib_ipc_get [](http://example.com/codeflow?start=23&length=15)

After some discussions offline, we decided that we don't need get and set APIs. So I'll remove those.